### PR TITLE
feat: add amoy, linea-sepolia, remove mumbai

### PR DESCRIPTION
**Pull request by nksazonov**
TODO:

- [x] Change addresses in `peers.json` in both amoy (80002) and linea-sepolia (59141).
- [x] Change all AA addresses in `aa.json` in linea-sepolia (59141).
- [x] Remove linea-goerli (59140).

### CODE REVIEW

File: networks/59141/markets.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[
+  "spot://ETH/USDC",
+  "spot://WETH/USDC",
+  "spot://WBTC/USDC",
+  "spot://DUCKIES/USDC",
+  "spot://USDC/USDT"
```
Comment (philanton) [inline networks/59141/markets.json:6]: I think we can add lube as well

File: networks/80002/markets.json
Comment (philanton) [inline networks/80002/markets.json:1]: Let's do only 5th and duckies for now

File: networks/59141/markets.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+[
+  "spot://ETH/USDC",
+  "spot://WETH/USDC",
```
Comment (philanton) [inline networks/59141/markets.json:3]: we don't need it, as we have etc

File: networks/59141/markets.json (hunk 1 above)
Comment (nksazonov) [reply networks/59141/markets.json]: badbcb5cbf000eadc1f7e12992055a838db1ef84

File: networks/80002/markets.json
Comment (nksazonov) [reply networks/80002/markets.json:1]: badbcb5cbf000eadc1f7e12992055a838db1ef84

File: networks/59141/markets.json (hunk 2 above)
Comment (nksazonov) [reply networks/59141/markets.json]: badbcb5cbf000eadc1f7e12992055a838db1ef84

File: networks/80002/markets.json [hunk 3]
```diff
@@ -1,9 +1,4 @@
 [
-    "spot://DUCKIES/USD",
-    "spot://USDT/USD",
-    "spot://MATIC/USD",
-    "spot://WBTC/USD",
     "spot://WETH/USD",
```
Comment (philanton) [inline networks/80002/markets.json:2]: oh, I think matic should be here, and in assets it should be with zero address

Comment (nksazonov) [reply networks/80002/markets.json]: be5bb6a0fe67790fb92eec93789c01a6e45e7ceb

**Review by philanton — APPROVED** [review]

**Pull request merged**